Microsoft.Toolkit.Uwp.UI.Animations/Extensions/AnimationExtensions.cs

```diff
@@ -87,7 +87,7 @@ public static EasingFunctionBase GetEasingFunction(EasingType easingType)
                 case EasingType.Sine:
                     return new SineEase();
                 default:
-                    throw new ArgumentOutOfRangeException(nameof(easingType), easingType, null);
+                    throw new NotSupportedException($"{nameof(easingType)} not supported");
```

Comment:
This should state what value is not supported. As is it will report `easingType not supported` When we should report something like `EasingType SomeNewValue, is not currently supported.`

Reply:
Good point, fixed.